test/blackbox-tests/test-cases/pkg/lockfile-generation.t/mock-opam-repository/packages/priorities/priorities.0.0.1/opam

```diff
@@ -0,0 +1,4 @@
+opam-version: "2.0"
+depends: [
+    "bar" & "quux" | "baz"
```

Comment:
Is this equivalent to `depends: [ "bar" ("quux" | "baz") ]` if so, I'm not sure if it's testing anything additional.

It should be something like: `(foo & bar) | baz` so that it can't be simplified.

Reply:
Yes, just simple terms listed one after another automatically get an `&`.